enforcer/src/main/java/org/wso2/micro/gateway/enforcer/grpc/ExtAuthService.java

```diff
@@ -52,21 +52,38 @@ public void check(CheckRequest request, StreamObserver<CheckResponse> responseOb
     }
 
     private CheckResponse buildResponse(ResponseObject responseObject) {
-        if (responseObject.getStatusCode() != 200) {
+        DeniedHttpResponse.Builder responseBuilder = DeniedHttpResponse.newBuilder();
+        HttpStatus status = HttpStatus.newBuilder().setCodeValue(responseObject.getStatusCode()).build();
+        if (responseObject.isDirectResponse()) {
+            // To handle options request
+            if (responseObject.getStatusCode() == 204) {
```

Comment:
Use the constant

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/1560/commits/264d082eae3bd5a5add1b0c913ac403de24b92a8

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/1560/commits/264d082eae3bd5a5add1b0c913ac403de24b92a8